nemo_text_processing/text_normalization/zh/verbalizers/char.py

```diff
@@ -0,0 +1,14 @@
+import pynini
+from nemo_text_processing.text_normalization.zh.graph_utils import NEMO_CHAR, NEMO_SIGMA, GraphFst,NEMO_NOT_SPACE,NEMO_NOT_QUOTE
+from pynini.lib import pynutil
+from nemo_text_processing.text_normalization.zh.utils import get_abs_path
+
+class Char(GraphFst):
```

Comment:
Use `name:`  field instead of `char` like in https://github.com/mzxcpp/NeMo/blob/main/nemo_text_processing/text_normalization/en/taggers/word.py#L90